src/hooks/engine.rs

```diff
@@ -154,7 +168,7 @@ pub static Host_InitializeGameDLL: Pointer<unsafe extern "C" fn()> = Pointer::em
     // Alternatively, find LoadEntityDLLs() and go to the parent function.
     Patterns(&[
         // 6153
-        pattern!(E8 ?? ?? ?? ?? 8B 0D ?? ?? ?? ?? 33 C0),
+        pattern!(E8 ?? ?? ?? ?? 8B 0D ?? ?? ?? ?? 33 C0 83 F9 01),
```

Comment:
Was it non-unique in 4554?

Reply:
yes, it was

Reply:
Interesting, I wonder what the other function was.

Reply:
one was `SV_WriteClientdataToMessage` and the other one was `VideoMode_SwitchMode`, this is what it matched for the latter one 
![image](https://user-images.githubusercontent.com/50155487/106754259-cfc89780-663d-11eb-82ee-05a1e561933e.png)

Reply:
I see, okay.